examples/Makefile

```diff
@@ -1,41 +1,26 @@
-JULIAHOME := $(abspath ..)
+SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+BUILDDIR := .
+JULIAHOME := $(abspath $(SRCDIR)/..)
+prefix := $(JULIAHOME)/usr
```

Comment:
We should decide whether this gets installed or not. If not, and this test should only ever run in a source-build tree and not on a binary install, then we'd need to move the execution of the embedding test to either before `mv julia julia2` or after `mv julia2 julia`, so that the source tree is still in place